src/lib.rs

```diff
@@ -58,9 +58,16 @@ pub mod frames;
 pub mod user_data;
 
 #[derive(Debug)]
+#[cfg_attr(feature = "serde_support", derive(serde::Serialize))]
+#[cfg_attr(
+    feature = "serde_support",
+    derive(serde::Deserialize),
+    serde(bound(deserialize = "'de: 'a"))
+)]
```

Comment:
These can probably be the same cfg_attr. You don't need a separate one for each derive.
```rust
#[cfg_attr(
    feature = "serde_support",
    derive(serde::Serialize, serde::Deserialize),
    serde(bound(deserialize = "'de: 'a"))
)]
```

Reply:
this was not possible